website/templates/project/project.mako

```diff
@@ -238,14 +238,25 @@
                    <a href="${node['url']}files/"> <i class="fa fa-external-link"></i> </a>
                 </div>
             </div>
-            <div class="panel-body">
-                <div id="treeGrid">
-                    <div class="spinner-loading-wrapper">
-                        <div class="logo-spin logo-lg"></div>
-                         <p class="m-t-sm fg-load-message"> Loading files...  </p>
+            % if not node['is_registration'] and not node['anonymous'] and 'write' in user['permissions']:
+                <div class="row">
+                    <div class="col-sm-12 m-t-sm m-l-md">
+                        <span class="f-w-xl">Click on a storage provider or drag and drop to upload</span>
                     </div>
                 </div>
-            </div>
+               <div class="panel-body panel-body-with-instructions">
+            %else:
+               <div class="panel-body">
+            %endif
+                    <div id="treeGrid">
+                        <div class="spinner-loading-wrapper">
+                            <div class="logo-spin logo-lg"></div>
+                             <p class="m-t-sm fg-load-message"> Loading files...  </p>
+                        </div>
+                    </div>
+                </div>
```

Comment:
Minor: for readability, it helps to add a comment like

```html
</div><!-- end .panel-body -->
```

especially because the opening tag is within an if.

 I'll add this when I merge.